alchemist-physical-agents/src/main/kotlin/it/unibo/alchemist/model/implementations/actions/physicalstrategies/Sum.kt

```diff
@@ -29,6 +29,6 @@ class Sum<T>(
 ) : PhysicalSteeringStrategy<T, Euclidean2DPosition, Euclidean2DTransformation, Euclidean2DShapeFactory> {
 
     override fun computeNextPosition(overallIntentionalForce: Euclidean2DPosition): Euclidean2DPosition =
-        (node.asProperty<T, PhysicalPedestrian2DProperty<T>>().physicalForces(environment) + overallIntentionalForce)
+        (node.asProperty<T, PhysicalPedestrian2D<T>>().physicalForces(environment) + overallIntentionalForce)
```

Comment:
nope. Save the property in the constructor, there is no need to recompute it over and over